navigator_api/main.py

```diff
@@ -38,98 +26,111 @@ def user_details():
+        },
+        {
+            "id": "dataset-3",
+            "organisation_name": "Antarctica",
+            "name": "Antarctica Inputs UNAIDS Estimates 2021"
+        },
+        {
+            "id": "dataset-4",
+            "organisation_name": "Sudan",
+            "name": "Sudan Inputs UNAIDS Estimates 2021"
         }
     )
 
 
-@main_blueprint.route('/dataset/<dataset_id>/state', methods=['GET'])
-@login_required
-def dataset_state_get(dataset_id):
-    logging.info(f'dataset_id: {dataset_id}')
-    possible_milestones = ['Shiny90', 'Naomi', 'Data Prep']
-    possible_tasks = [
-        "Generate input data",
-        "Go to Spectrum/AIM. Row 34",
-        "Download data templates from ADR",
-        "Populate Shiny90 survey template",
-        "Populate Shiny90 HIV testing template?",
-        "Populate the Population template",
-        "Populate ANC survey template",
-        "Populate ART template",
-        "Populate Naomi survey template",
-        "Populate geo template",
-        "Create user account",
-    ]
-    task_number = randint(2, 85)
-    task_name = possible_tasks[
-        randint(0, len(possible_tasks)-1)
-    ]
-    list_items = ('').join([
-        f'<li>{get_sentence(count=(1, 3))}</li>'
-        for x in range(randint(1, 7))
-    ])
-    html = ''.join([
-        f'<h3>{task_name}</h3>',
-        f'<p>{get_sentence(count=(1, 3))}</p>',
-        f'<ol>{list_items}</ol>'
-    ])
-    actionable_links = [
-        {
-            'label': possible_tasks[
-                randint(0, len(possible_tasks)-1)
-            ],
-            'link': 'http://example.com'
+@main_blueprint.route('/workflows')
+def workflow_list():
+    return jsonify({
+        "workflows": [
+            {
+                "id": "dataset-1",
+                "name": "Uganda Inputs UNAIDS Estimates 2021"
+            },
+            {
+                "id": "dataset-3",
+                "name": "Antarctica Inputs UNAIDS Estimates 2021"
+            }
+        ]
+    })
+
+
+@main_blueprint.route('/workflows/<dataset_id>/state')
+def workflow_state(dataset_id):
+    return jsonify({
+        "id": "xxx-yyy-zzz",
+        "milestones": [
+            {
+                "id": "xxx",
+                "title": "Naomi Data Prep",
+                "completed": True,
+                "progress": 100
+            },
+            {
+                "id": "yyy",
+                "title": "Shiny 90 Data Prep",
+                "completed": False,
+                "progress": 50
+            },
+            {
+                "id": "zzz",
+                "title": "Update Spectrum",
+                "completed": False,
+                "progress": 0
+            }
+        ],
+        "tasks": [
+            "aaabbbbcc",
+            "dddeeefff",
+            "asdfasdfa",
+            "aaabbbbcc",
+            "dddeeefff",
+            "asdfasdfa",
+            "aaabbbbcc",
+            "dddeeefff",
+            "asdfasdfa"
```

Comment:
@tomeksabala Is this a list of task ids? Why do we have this?

Reply:
That's the link of previous tasks user has visited.